apps/frontend/src/pages/root/components/user-wheel/user-wheel.tsx

```diff
@@ -21,7 +26,7 @@ const UserWheel: React.FC = () => {
 	const dispatch = useAppDispatch();
 	const { dataStatus, scores } = useAppSelector((state) => state.quiz);
 	const [isEditingModalOpen, setIsEditingModalOpen] = useState<boolean>(false);
-
+	const [editMode, setEditMode] = useState<WheelEditMode>("manual");
```

Comment:
Maybe it would be better to create `WheelEditMode` as enum to store modes and use `ValueOf <typeof WheelEditMode>` to set type?

Reply:
I also thought of that, but it felt like too much for a single switch component inside of a single file